src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -46,11 +69,25 @@ static DicomWebClient()
 #pragma warning restore CS0618 // Type or member is obsolete
         }
 
-        public DicomWebClient(HttpClient httpClient)
+        public DicomWebClient(Uri baseAddress, HttpMessageHandler httpMessageHandler)
         {
-            HttpClient = httpClient;
-            _jsonSerializerSettings = new JsonSerializerSettings();
-            _jsonSerializerSettings.Converters.Add(new JsonDicomConverter(writeTagsAsKeywords: true));
+            HttpClient = new HttpClient(new DicomWebResponseHandler(httpMessageHandler))
```

Comment:
Generally not recommended to create a new HttpClient. If they have a singleton DicomWebClient this is fine, otherwise the recommendation is to use HttpClientFactory to get a instance.

Also the initialization of this in Dicom-cast, is that working still with these changes?

Reply:
Reverted the change with enhancement.